.travis.yml

```diff
@@ -7,5 +7,6 @@ install:
 
 script:
   - npm run unit
-  - npm run posttest
+  - npm run post-unit
   - npm run integration
+  - npm run post-integration
```

Comment:
The last line can be 
`  - npm run post-integration || :`

We need the extra bit to make sure that this command does not fail on travis.

Reply:
This failed only in the travis build during PR. It worked fine in my commit.

Reply:
Do I need to include `:` too?

Reply:
Yes. The ":" is essential. It is equivalent to "no operation" which always succeeds.